src/components/ShoppingCart/CartListItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Button, ButtonGroup, Flex, Td, Tr, Text } from "@chakra-ui/react";
+import React from "react";
+import { FaPen } from "react-icons/fa";
+import { MdDeleteForever } from "react-icons/md";
+import { ICartItem } from "@/interfaces/cartItem";
+
+const CartListItem = (props: ICartItem) => {
+  // import cart information
+  const productName = props.design.designName;
+  const quotation = props.quotation;
+  const quotationDetails = props.quotationDetails;
+  const image = props.previewPic;
+
+  return (
+    <>
+      <Tr alignItems="center" role="dataRow">
+        <Td padding="20px 60px" sx={{ "vertical-align": "top" }} height="180px">
+          {/* preserve space for the Court image */}
```

Comment:
You can use sentences like
Todo: space for thumbnail images that implementing in the future.

Todo is the word for something that will need to be done in the future so that global search on IDE by using this word helps team members know where is the place to start later.

Reply:
> You can use sentences like Todo: space for thumbnail images that implementing in the future.
> 
> Todo is the word for something that will need to be done in the future so that global search on IDE by using this word helps team members know where is the place to start later.

The code comments was changed to {/* Todo: space for thumbnail images that implementing in the future.*/}